.github/workflows/CI.yml

```diff
@@ -529,8 +529,8 @@ jobs:
         if: steps.e2etest-release.outputs.cache-hit != 'true'
         run: |
           docker load < build/packages/$IMAGE_NAME.tar
```

Comment:
BTW, I'd like to remove docker load from the workflow - we should just push to the ECR repo instead of docker saving. Then we could use the official docker build github action, using buildx, and making arm64 build trivial

Reply:
@khanhntd imo the repo should be named adot-collector-tests ; we could potentially have other test images there too wdty?

Reply:
True thought. Mostly I thought the docker saving there is for separating the e2e preparations part with the build parts. Other than that, I don't see any reasons to use ```docker save``` so I am down with using docker buildx for arm64.

Reply:
@alolita the only two tests that we have are: integration test from the CI and validation test from the CD. But the CD release is using ```public.ecr.aws/aws-observability/aws-otel-collector```; so the only one we are using for this repo is the CI. So IMO, if we are planning to add additional testing such as unit testing or config testing such as FB in the future, then we should go with yours; if we don't, then specific enough such as ```adot-collector-integration-test``` is fine since its only for the CI.

Reply:
However, such tests are not feasible since we are a downstream repo so I cannot imagine we have addition tests beside integration test such as unit test or config test.